ci/images/e2e-test-runner/run-e2e-test.sh

```diff
@@ -0,0 +1,75 @@
+#!/usr/bin/env bash
+set -o errexit
+set -o nounset
+set -o pipefail
+set -x
+
+SCRIPT_DIR="$(
+  cd "$(dirname "$0")" >/dev/null
+  pwd
+)"
+
+PROJECT_DIR="$(
+  cd "$SCRIPT_DIR/../../.." >/dev/null || exit 1
+  pwd
+)"
+
+# shellcheck source=ci/images/ci-runner/hack/bin/utils.sh
+source "$PROJECT_DIR/ci/images/ci-runner/hack/bin/utils.sh"
+
+export GITHUB_USER GITHUB_TOKEN QUAY_TOKEN QUAY_OAUTH_USER QUAY_OAUTH_TOKEN QUAY_OAUTH_TOKEN_RELEASE_SOURCE QUAY_OAUTH_TOKEN_RELEASE_DESTINATION \
+       GITHUB_ACCOUNTS_ARRAY PREVIOUS_RATE_REMAINING GITHUB_USERNAME_ARRAY GH_RATE_REMAINING
+
+printf "Fetch secrets from bitwarden server\n" | indent 2
+open_bitwarden_session
+get_github_user
+get_github_token
+get_quay_token
+get_quay_oauth_user
+get_quay_oauth_token
+get_quay_oauth_token_release_source
+get_quay_oauth_token_release_destination
+get_github_accounts
+
+
+PREVIOUS_RATE_REMAINING=0
+
+# user stored: username:token,username:token
+for account in "${GITHUB_ACCOUNTS_ARRAY[@]}"
+do :
+    IFS=':' read -r -a GITHUB_USERNAME_ARRAY <<< "$account"
+
+    GH_RATE_REMAINING=$(curl -s \
+    -H "Accept: application/vnd.github+json" \
+    -H "Authorization: Bearer ${GITHUB_USERNAME_ARRAY[1]}"\
+    https://api.github.com/rate_limit | jq ".rate.remaining")
+
+    echo -e "[INFO ] user: ${GITHUB_USERNAME_ARRAY[0]} with rate limit remaining $GH_RATE_REMAINING"
+    if [[ "${GH_RATE_REMAINING}" -ge "${PREVIOUS_RATE_REMAINING}" ]];then
+        GITHUB_USER="${GITHUB_USERNAME_ARRAY[0]}"
+        GITHUB_TOKEN="${GITHUB_USERNAME_ARRAY[1]}"
+    fi
+    PREVIOUS_RATE_REMAINING="${GH_RATE_REMAINING}"
+done
+
+echo -e "[INFO] Start tests with user: ${GITHUB_USER}"
+
+##KUBECONFIG
```

Comment:
That's seem related to git, not kubernetes.

Reply:
fixed.